apis/imdb.py

```diff
@@ -1,7 +1,19 @@
 import requests
 from bs4 import BeautifulSoup, SoupStrainer
-from constants import movieDB_key, imdb_api_url
+from constants import movieDB_key, imdb_api_url, wikipedia_url
 from typing import Dict
+from apis.wikipedia import get_short_details
+
+
+def get_data_from_wiki(url: str) -> Dict:
+    return get_short_details(url)

```

Comment:
why not just us `get_short_details` instead having a wrapper function top of it

Reply:
I'm gonna add more code in the function after we've finalized what data we need and what we can discard from Wikipedia. Waiting for Joel or rj to finalize that part. I think then this function would be more relevant.

Reply:
Sounds good, we'll put this on our plate for next week. Hoping to do the integration next week to get Mercury in production!